src/pages/Lending/hooks/useLendingCloseQuery.ts

```diff
@@ -95,21 +107,30 @@ const selectLendingCloseQueryData = memoize(
       bn(quoteOutboundDelayMs).plus(quoteInboundConfirmationMs),
     ).toNumber()
 
+    const repaymentAmountFiatUsd = bnOrZero(repaymentAmountFiatUserCurrency)
+      .times(bn(1).div(userCurrencyToUsdRate))
```

Comment:
same for all these

Reply:
Might as well for cleanliness! ad3e79356b4c923b7c4c215792ec21f26a94b126